### DISCUSSION TITLE
How to change vehicle properties

### DISCUSSION THREAD

**Discussion by Monf1a** [openTCS/opentcs#23] [Q&A] [answered]
How to change vehicle properties, such as acceleration ?

**Comment by martingr** [comment, marked as answer]
Hi, the loopback adapter supports some configuration for virtual vehicle's, see [1]. For example, to set the acceleration of a virtual vehicle, set the `loopback:acceleration` property on the corresponding vehicle in the plant model.

[1]: https://www.opentcs.org/docs/5.5/user/opentcs-users-guide.html#_configuring_a_virtual_vehicles_characteristics

**Reply by Monf1a** [reply]
thanks there : )
